[ci] `lerna fatal: bad revision ''` Failed job on npm publish

Reference: https://github.com/layer5io/sistent/actions/runs/6426640807/job/17451161711

```bash
/home/runner/work/_temp/5cac6ec6-75c5-4b57-a685-98a4c38226fc.sh: line 4: [: =: unary operator expected
lerna notice cli v7.3.0
lerna info versioning independent
lerna info ci enabled
lerna WARN conventional-graduate all packages
lerna info Graduating all prereleased packages
lerna info Looking for changed packages since 
lerna ERR! Error: Command failed with exit code 128: git diff --name-only  -- apps/design-system
lerna ERR! fatal: bad revision ''
lerna ERR!     at makeError (/home/runner/work/sistent/sistent/node_modules/execa/lib/error.js:60:11)
lerna ERR!     at module.exports.sync (/home/runner/work/sistent/sistent/node_modules/execa/index.js:[19](https://github.com/layer5io/sistent/actions/runs/6426640807/job/17451161711#step:5:20)4:17)
lerna ERR!     at execSync (/home/runner/work/sistent/sistent/node_modules/@lerna/child-process/index.js:39:16)
lerna ERR!     at diffSinceIn (/home/runner/work/sistent/sistent/node_modules/lerna/dist/index.js:440:10)
lerna ERR!     at hasDiffSinceThatIsntIgnored (/home/runner/work/sistent/sistent/node_modules/lerna/dist/index.js:413:18)
lerna ERR!     at isCandidate (/home/runner/work/sistent/sistent/node_modules/lerna/dist/index.js:500:77)
lerna ERR!     at /home/runner/work/sistent/sistent/node_modules/lerna/dist/index.js:508:9
lerna ERR!     at Array.forEach (<anonymous>)
lerna ERR!     at collectProjects (/home/runner/work/sistent/sistent/node_modules/lerna/dist/index.js:507:12)
lerna ERR! lerna Command failed with exit code 1[28](https://github.com/layer5io/sistent/actions/runs/6426640807/job/17451161711#step:5:29): git diff --name-only  -- apps/design-system
lerna ERR! lerna fatal: bad revision ''
Error: Process completed with exit code 128.
```

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
  - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -62,7 +62,7 @@ jobs:
             if [ ${{ github.base_ref }} = development ]; then
             npx lerna version --no-private --conventional-commits --conventional-prerelease --preid beta --yes
             else
-            npx lerna version --no-private --conventional-commits --conventional-graduate --yes
+            npx lerna version --no-private --conventional-commits --yes
             fi
             
             npx lerna publish from-git --yes

